### ISSUE TITLE
Quality Armory Vehicles Conflict

### ISSUE THREAD

**Issue by Prowco** [PikaMug/Quests#924]
**What is your Quests version / build number (do _not_ say "latest")?**
3.7.8
**Describe the problem and how to replicate it. Any console errors to post _using Pastebin.com_?**Well for some reason i dont understand if I am in a vehicle does not recognize when i reach a location objective, I need to get off the vehicle if i want quests recognize that it reaches the objective is there any way for fix this because I need recognize that I reached the location while being in a vehicle since the quests I am doing are in a city so I need recognize that I am in the vehicle.
**If applicable, can you provide an example quest from quests.yml _using Pastebin.com_?**
Errors i got in console:
https://pastebin.com/W2b2521z

**Comment by PikaMug** [comment]
Thanks for the report. Please confirm whether this still occurs on the latest dev build https://ci.codemc.org/job/PikaMug/job/Quests/

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Prowco** [comment]
Yes still Hapens here i take some screenshots so  u can see the error better
Here i am in a vehicle in a location objective as u can see quests doing nothing  happens i dont complete the objective
https://i.imgur.com/pmyhGaz.png
and here i  get off of the vehicle in the same place and  inmediatly i complete the objective reach location
https://i.imgur.com/yBdLwvE.png
You think u can fix this bc i need u can reach the locations inside of a vehicle..

**Comment by PikaMug** [comment]
@Prowco No, that’s just the API.

Please direct future questions to the issue I opened on the QualityArmory github.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
